neofetch

```diff
@@ -3549,16 +3549,26 @@ END
                 # like a font definition. NOTE: There is a slight limitation in this approach.
                 # Technically "Font Name" is a valid font. As it doesn't specify any font options
                 # though it is hard to match it correctly amongst the rest of the noise.
-                [[ -n "$binary" ]] &&
-                    term_font=$(
-                        strings "$binary" |
+                if [[ -n $(strings "$binary" | grep resource) ]]; then
```

Comment:
Can we save `strings "$binary"` to a variable so we don't call it twice in some cases?